test/integration/merkle-lockup/MerkleLockup.t.sol

```diff
@@ -24,12 +24,39 @@ abstract contract MerkleLockup_Integration_Test is Integration_Test {
     //////////////////////////////////////////////////////////////////////////*/
 
     function claimLL() internal returns (uint256) {
-        return merkleLL.claim({
-            index: defaults.INDEX1(),
-            recipient: users.recipient1,
-            amount: defaults.CLAIM_AMOUNT(),
-            merkleProof: defaults.index1Proof()
-        });
+        return claimLLAt(1);
+    }
+
+    function claimLLAt(uint256 index) internal returns (uint256) {
```

Comment:
we don't need this function

Reply:
How would you write tests in `clawback.t.sol`? Because now the test demands claims in consecutive order.

Reply:
see this: https://github.com/sablier-labs/v2-periphery/pull/340#discussion_r1588764811